src/lib/select/select.ts

```diff
@@ -194,7 +195,7 @@ export class MatSelectTrigger {}
     '[attr.aria-required]': 'required.toString()',
     '[attr.aria-disabled]': 'disabled.toString()',
     '[attr.aria-invalid]': 'errorState',
-    '[attr.aria-owns]': 'panelOpen ? _optionIds : null',
+    '[attr.aria-owns]': '_optionIds',
```

Comment:
If https://github.com/angular/material2/pull/7835 ends up being merged, it would be nice to be able to use `role="combobox"` in correlation with the editable textbox that is focus-trapped in the mat-select-header.

Reply:
The text input is optional for `combobox`; this prototype uses it without one:
https://stackblitz.com/edit/angular-a8j36n?file=app%2Fmat-select%2Fmat-select.component.html

As well as the deque select:
https://pattern-library.dequelabs.com/components/selects

Reply:
In aria 1.0 it's not required and in aria 1.1 textbox or searchbox is required. 

https://www.w3.org/TR/wai-aria-1.1/#combobox

Reply:
Ah, interesting. We'll have to explore using `combobox` for a select with a search input in the future